test/es-module/test-esm-invalid-data-urls.js

```diff
@@ -5,20 +5,14 @@ const assert = require('assert');
 (async () => {
   await assert.rejects(import('data:text/plain,export default0'), {
     code: 'ERR_INVALID_MODULE_SPECIFIER',
-    message:
-      'Invalid module "data:text/plain,export default0" has an unsupported ' +
-      'MIME type "text/plain"',
+    message: 'Invalid module "data:text/plain,export default0" has an unsupported MIME type "text/plain"',
```

Comment:
A recent change in the linter rules allow to have strings longer than 80 chars. I'd prefer if we leave this change out because:

- It's unrelated to JSON modules unflagging
- It might trigger a linter error when trying to backport to LTS branch (the linter change has not been backported)